src/renderer/index.js

```diff
@@ -4,9 +4,16 @@ import ReactDom from 'react-dom';
 
 import App from './app';
 import { ipcMainChannels } from '../main/ipcMainChannels';
+import { getSettingsValue } from './components/SettingsModal/SettingsStorage';
 
 const logger = window.Workbench.getLogger(__filename.split('/').slice(-1)[0]);
 
+const language = await getSettingsValue('language');
+
+// call this before rendering the app so that _() is defined
+ipcRenderer.invoke(ipcMainChannels.SET_LANGUAGE, language);
+window._ = ipcRenderer.sendSync.bind(null, ipcMainChannels.GETTEXT);  // partially applied function
```

Comment:
When I tried replacing `sendSync` with `send` or `invoke`, the strings wrapped in `_()` don't render at all:
<img width="879" alt="image" src="https://user-images.githubusercontent.com/43770515/151896539-ecbe2e80-302b-4eba-8589-f294aeba25f8.png">
I think because it's not `await`ing the result. And using async/await in `render` functions doesn't seem to be supported.

Reply:
Right, that makes sense. Okay, I guess we'll assume there's no harm in synchronous `_()` calls.